### ISSUE TITLE
README is incorrect about mixin path

### ISSUE THREAD

**Issue by jamesarosen** [DockYard/ember-validations#299]
The README says

> You need to mixin EmberValidations into any Ember.Object you want to add validations to:

```js
import Ember from 'ember';
import EmberValidations from 'ember-validations';

export default Ember.ObjectController.extend(EmberValidations);
```

`ember-validations/index.js` on my filesystem (v2.0.0-alpha.3) is
```js
export default {
  Mixin: Mixin,
  validator: function(callback) {
    return { callback: callback };
  }
};
```
which means it should be `.extend(EmberValidations.Mixin)`.

It looks like this changed recently, but the README isn't consistent with the latest release.

**Comment by danmcclain** [comment]
https://github.com/dockyard/ember-validations/issues/291#issuecomment-100498781

**Comment by albertpak** [comment]
@jamesarosen ran into same issue today

**Comment by martndemus** [comment]
I am going to close this issue for now, feel free to reopen this issue if you feel like it's still relevant and we should take a look at it.

**Comment by jamesarosen** [comment]
Yes, this is still relevant. The README says
```js
import Ember from 'ember';
import EmberValidations from 'ember-validations';
export default Ember.Controller.extend(EmberValidations);
```
but the correct usage is
```js
import Ember from 'ember';
import EmberValidations from 'ember-validations';
export default Ember.Controller.extend(EmberValidations.Mixin);
```

**Comment by martndemus** [comment]
@jamesarosen Can you verify the fix in #430 works for you?

**Comment by jamesarosen** [comment]
It sure does!
